### ISSUE TITLE
object detection - key error /m/01g317

### ISSUE THREAD

**Issue by alowc** [tensorflow/models#9374]
# Prerequisites

Please answer the following questions for yourself before submitting an issue.

- [x] I am using the latest TensorFlow Model Garden release and TensorFlow 2.
- [x ] I am reporting the issue to the correct repository. (Model Garden official or research directory)
- [ ] I checked to make sure that this issue has not been filed already.

## 1. The entire URL of the file you are using

https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/tf2_detection_zoo.md

## 2. Describe the bug

I'm trying to run a training procedure on the pre-trained network CenterNet Resnet50 V1 FPN 512x512, available in the above mentioned repository.

## 3. Steps to reproduce

**MODELS_CONFIG = {
'CenterNet-Resnet50-V1-FPN': {
        'model_name': 'centernet_resnet50_v1_fpn_512x512_coco17_tpu-8',
        'base_pipeline_file': 'centernet_resnet50_v1_fpn_512x512_kpts_coco17_tpu-8.config',
        'pretrained_checkpoint': 'centernet_resnet50_v1_fpn_512x512_coco17_tpu-8.tar.gz',
        'batch_size': 16
    },
    'CenterNet-Resnet50-V1-FPN-Keypoints': {
        'model_name': 'centernet_hg104_512x512_kpts_coco17_tpu-32',
        'base_pipeline_file': 'centernet_resnet50_v1_fpn_512x512_kpts_coco17_tpu-8.config',
        'pretrained_checkpoint': 'centernet_hg104_512x512_kpts_coco17_tpu-32.tar.gz',
        'batch_size': 16
    }
}

chosen_model = 'CenterNet-Resnet50-V1-FPN'

num_steps = 40000 #The more steps, the longer the training. Increase if your loss function is still decreasing and validation metrics are increasing. 
num_eval_steps = 500 #Perform evaluation after so many steps

model_name = MODELS_CONFIG[chosen_model]['model_name']
pretrained_checkpoint = MODELS_CONFIG[chosen_model]['pretrained_checkpoint']
base_pipeline_file = MODELS_CONFIG[chosen_model]['base_pipeline_file']
batch_size = MODELS_CONFIG[chosen_model]['batch_size'] #if you can fit a large batch in memory, it may speed up your training**

#download pretrained weights
%mkdir /content/drive/My\ Drive/object_detection/deploy
%cd /content/drive/My\ Drive/object_detection/deploy
import tarfile
download_tar = 'http://download.tensorflow.org/models/object_detection/tf2/20200711/' + pretrained_checkpoint

!wget {download_tar}
tar = tarfile.open(pretrained_checkpoint)
tar.extractall()
tar.close()

#download base training configuration file
%cd /content/drive/My\ Drive/object_detection/deploy
download_config = 'https://raw.githubusercontent.com/tensorflow/models/master/research/object_detection/configs/tf2/' + base_pipeline_file
!wget {download_config}

#prepare
pipeline_fname = '/content/drive/My Drive/object_detection/deploy/' + base_pipeline_file
fine_tune_checkpoint = '/content/drive/My Drive/object_detection/deploy/' + model_name + '/checkpoint/ckpt-0'

def get_num_classes(pbtxt_fname):
    from object_detection.utils import label_map_util
    label_map = label_map_util.load_labelmap(pbtxt_fname)
    categories = label_map_util.convert_label_map_to_categories(
        label_map, max_num_classes=90, use_display_name=True)
    category_index = label_map_util.create_category_index(categories)
    return len(category_index.keys())
num_classes = get_num_classes(label_map_pbtxt_fname)

...

pipeline_file = '/content/drive/My\ Drive/object_detection/deploy/pipeline_file.config'
model_dir = '/content/training/'

**!python /content/models/research/object_detection/model_main_tf2.py \
    --pipeline_config_path={pipeline_file} \
    --model_dir={model_dir} \
    --alsologtostderr \
    --num_train_steps={num_steps} \
    --sample_1_of_n_eval_examples=1 \
    --num_eval_steps={num_eval_steps}**

## 4. Expected behavior

The training procedure is expected to start.

## 5. Additional context

key error /m/01g317

## 6. System information

**Using _Google Colab_ virtual environment.**

**Comment by orihash** [comment]
Maybe you should try changing the pipeline.config **keypoint_class_name** value to from "/m/01g317" to "person" if you are training for person, like on your keypoints labels map **name**: "person", that worked for me.

**Comment by jvishnuvardhan** [comment]
@alowc This is a stale issue. Is this still an issue for you. Can you please check with recent TF versions and let us know whether the issue persists with recent TF versions or not. [Here](https://colab.research.google.com/gist/jvishnuvardhan/d7fe3ca280bfc72de8b441b851115d3a/untitled1207.ipynb) is a gist for our reference. Thanks!
